pkg/container/resolver_test.go

```diff
@@ -163,33 +154,35 @@ func TestResolverLocalManifest(t *testing.T) {
 		".",
 	)
 
+	cmd.Stderr = os.Stderr
+	cmd.Stdout = os.Stdout
+
 	err = cmd.Run()
-	if err != nil {
-		t.Fatalf("Error building podman containers: %v", err)
-	}
+	assert.NoError(t, err)
 
+	local := true
 	// try resolve an x86_64 container using a local manifest list
 	resolver := container.NewResolver("amd64")
 	resolver.Add(container.SourceSpec{
 		"localhost/multi-arch",
 		"",
 		common.ToPtr(""),
 		common.ToPtr(false),
-		true,
+		local,
 		&tmpStorage,
 	})
 	specs, err := resolver.Finish()
 	assert.NoError(t, err)
 	assert.Len(t, specs, 1)
```

Comment:
It was actually really difficult to try and assert that the expected id matches.

Reply:
Does it change?  I wonder what the problem is.  Maybe we can have tiny container archives checked in to the repo so they never change and we can `podman pull` them into the registry to create the manifest with known container IDs.